homeassistant/components/light/hue.py

```diff
@@ -243,33 +268,44 @@ def unique_id(self):
 
     @property
     def name(self):
-        """Return the mame of the Hue light."""
+        """Return the name of the Hue light."""
         return self.info.get('name', DEVICE_DEFAULT_NAME)
 
     @property
     def brightness(self):
         """Return the brightness of this light between 0..255."""
-        return self.info['state'].get('bri')
+        if self.is_group:
+            return self.info['action'].get('bri')
+        else:
+            return self.info['state'].get('bri')
 
     @property
     def xy_color(self):
         """Return the XY color value."""
-        return self.info['state'].get('xy')
+        if self.is_group:
+            return self.info['action'].get('xy')
+        else:
+            return self.info['state'].get('xy')
 
     @property
     def color_temp(self):
         """Return the CT color value."""
-        return self.info['state'].get('ct')
+        if self.is_group:
+            return self.info['action'].get('ct')
+        else:
+            return self.info['state'].get('ct')
 
     @property
     def is_on(self):
         """Return true if device is on."""
-        self.update_lights()
-
-        if self.allow_unreachable:
-            return self.info['state']['on']
+        if self.is_group:
+            return self.info['state']['any_on']
         else:
-            return self.info['state']['reachable'] and self.info['state']['on']
+            if self.allow_unreachable:
+                return self.info['state']['on']
+            else:
+                return self.info['state']['reachable'] and \
+                       self.info['state']['on']
```

Comment:
Hmm, no idea how the spacing should be. `script/lint_docker` was happy with this indentation.